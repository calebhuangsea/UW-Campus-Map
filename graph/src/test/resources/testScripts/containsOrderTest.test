# Test contain order in three nodes file
# Tests that a graph can be created
# three nodes can be added to the graph
# contain order is correct

# Create a graph
CreateGraph graph1

# Add three nodes
AddNode graph1 n3
AddNode graph1 n2
AddNode graph1 n1

# Print the nodes in the graph
ListNodes graph1