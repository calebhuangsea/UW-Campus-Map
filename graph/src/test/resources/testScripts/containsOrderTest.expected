# Test contain order in three nodes file
# Tests that a graph can be created
# three nodes can be added to the graph
# contain order is correct

# Create a graph
created graph graph1

# Add three nodes
added node n3 to graph1
added node n2 to graph1
added node n1 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2 n3